<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mw_gestaoVendas</class>
 <widget class="QDialog" name="mw_gestaoVendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateEdit" name="de_dataini">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>39</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>9</month>
     <day>14</day>
    </datetime>
   </property>
  </widget>
  <widget class="QDateEdit" name="de_datafim">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>39</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>16</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Inicial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>16</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Final</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_filtrar">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>38</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_relatorioPDF">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>460</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar Relat√≥rio em PDF</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listaVendas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>611</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listaProdutosVendas">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>200</y>
     <width>611</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mostrarTodasVendas">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>37</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar todas vendas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
